python/ray/autoscaler/_private/providers.py

```diff
@@ -136,6 +149,7 @@ def _import_external(provider_config):
 
 _PROVIDER_PRETTY_NAMES = {
     "readonly": "Readonly (Manual Cluster Setup)",
+    "fake_docker": "Fake Docker",
```

Comment:
Can this be a sub option of fake multinode instead of entirely separate? It seems the logic is mostly the same except the docker option.